pyfr/readers/base.py

```diff
@@ -22,3 +26,209 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class BaseMeshConverter(object):
```

Comment:
I don't think so. Face numbering (_petype_fnums) is only related with PyFR mesh file format. I think this numbering is associated with faces of Shape class in shape.py.